gulpfile.js

```diff
@@ -131,24 +151,81 @@ gulp.task('buildApps', function() {
     );
 });
 
+gulp.task('build-specs', function buildSpecs() {
+    var cesiumViewerOutputDirectory = 'Build/Apps/CesiumViewer';
+    mkdirp.sync(cesiumViewerOutputDirectory);
+
+    var promise = Promise.join(
+        rollup.rollup({
+            input: 'Specs/SpecList.js',
+            plugins: [
+                rollupPluginExternalGlobals({
+                    '../Source/Cesium.js': 'Cesium',
+                    '../../Source/Cesium.js': 'Cesium',
+                    '../../../Source/Cesium.js': 'Cesium',
+                    '../../../../Source/Cesium.js': 'Cesium'
```

Comment:
Can we just declare this object once at the top of this function and re-use it, since it seems to be copied 3 times here.

Reply:
Also, this doesn't seem like it's supposed to be in the `build-specs` function?

```
var cesiumViewerOutputDirectory = 'Build/Apps/CesiumViewer';
    mkdirp.sync(cesiumViewerOutputDirectory);
```

Reply:
Thanks, means to do some self-review of this file and I guess I missed them.  Breaking up gulpfile.js and doing a big clean-up is pretty high on my list of side projects after this is merged.